var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -83,11 +83,13 @@ class Openmpi(AutotoolsPackage):
     # Still supported
     version('4.0.1', sha256='cce7b6d20522849301727f81282201d609553103ac0b09162cf28d102efb9709')  # libmpi.so.40.20.1
     version('4.0.0', sha256='2f0b8a36cfeb7354b45dda3c5425ef8393c9b04115570b615213faaa3f97366b')  # libmpi.so.40.20.0
+    version('3.1.5', sha256='fbf0075b4579685eec8d56d34d4d9c963e6667825548554f5bf308610af72133')  # libmpi.so.40.10.4
```

Comment:
Should we make this the new preferred version?

Reply:
no,you should make 4.0.2 the preferred version.  Last time that was suggested it caused a stir in the Spack community.

Reply:
Well, upgrading from OpenMPI 3.X to 4.X is a much bigger deal than upgrading from 3.1.4 to 3.1.5. I don't actually use MPI, so it's not up to me 😛

Reply:
I'd suggest just leaving preferred like it is for now.